targets/frontend/src/modules/authentification/deleteUser.ts

```diff
@@ -12,28 +12,41 @@ const deleteQuery = gql`
         isDeleted: true
       }
     ) {
-      name
+      id
     }
   }
 `;
 
 interface DeleteUserHasuraResult {
   update_auth_users_by_pk: {
-    name: string;
+    id: string;
   };
 }
 
-export const deleteUser = async (userId: string): Promise<boolean> => {
+const anonymizeUser = (userName: string, userId: string): string => {
+  if (!userName?.length) return userId.slice(4);
+  let anonymous = userName[0].toUpperCase();
+  const spaceIndex = userName.indexOf(" ");
+  if (spaceIndex > 0 && userName.length > spaceIndex) {
+    anonymous += userName[spaceIndex + 1].toUpperCase();
+  }
+  return anonymous;
```

Comment:
```suggestion
  return userName.toUpperCase().split(" ").map((word) => word[0]).join();
```

Version moins procédurale :)